towhee/data/image_datasets.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import pandas as pd
+from pandas import Series
+from torch.utils.data import Dataset
+from PIL import Image
+
+
+class ImageDataset(Dataset):
+    """
+        PytorchImageDataset is a dataset class for PyTorch.
```

Comment:
TorchImageDataset or ImageDataset?
There are two different names in class declaration and docstring